src/jax_loop_utils/metric_writers/mlflow/metric_writer.py

```diff
@@ -79,6 +81,10 @@ def __init__(
                 experiment_id = experiment.experiment_id
         self._run_id = self._client.create_run(experiment_id=experiment_id, run_name=run_name).info.run_id
 
+    def set_tags(self, tags: dict[str, Any], synchronous: Optional[bool] = None) -> Optional[RunOperations]:
```

Comment:
Try to be somewhat consistent with the other method interfaces.

```suggestion
    def write_tags(self, tags: dict[str, Any]):
```

Reply:
It was consistent with the mlflow interface. Why should this MLFlow specific method look like a metric writer method?